components/Frame/Popover/NavLink.js

```diff
@@ -45,6 +47,20 @@ export const NavA = React.forwardRef(
           }
         }))}
       {...(inline ? styles.inline : styles.block)}
+      {...(large &&
+        css({
+          '& + &': {
+            marginTop: 24
+          },
+          ...fontStyles.sansSerifMedium20,
+          [mediaQueries.mUp]: {
+            ...fontStyles.sansSerifMedium22,
+            '& + &': {
+              marginLeft: 36,
+              marginTop: 0
+            }
+          }
+        }))}
```

Comment:
Move this to `styles` as a `large` key and then do `styles.large` here. Currently on every render of every instance it has to compute a css hash and then realise that it already generated the css rule.